bazel/repository_locations.bzl

```diff
@@ -704,6 +704,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         extensions = ["envoy.wasm.runtime.wavm"],
         cpe = "cpe:2.3:a:webassembly_virtual_machine_project:webassembly_virtual_machine:*",
     ),
+    com_github_wamr = dict(
+        project_name = "Webassembly Micro Runtime",
+        project_desc = "A standalone runtime with a small footprint for WebAssembly",
+        project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
+        version = "0.1-alpha",
+        sha256 = "e364c84654121e3b303d380e017540bb1b330845327db3f4d51fa353492ec04f",
+        strip_prefix = "wasm-micro-runtime-{version}",
+        urls = ["https://github.com/lum1n0us/wasm-micro-runtime/archive/refs/tags/v{version}.tar.gz"],
```

Comment:
Why not https://github.com/bytecodealliance/wasm-micro-runtime?

Reply:
The changes will be merged into https://github.com/bytecodealliance/wasm-micro-runtime after finish the envoy checking.